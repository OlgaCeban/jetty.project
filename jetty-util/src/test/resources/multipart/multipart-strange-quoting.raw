--tR61vgdxSzzv2FDycET2lt-OUZ1IW1GqA
Content-Disposition: form-data; name="and \"I\" quote"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 1
--tR61vgdxSzzv2FDycET2lt-OUZ1IW1GqA
Content-Disposition: form-data; name="and+%22I%22+quote"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 2
--tR61vgdxSzzv2FDycET2lt-OUZ1IW1GqA
Content-Disposition: form-data; name="value\"; what=\"whoa\""
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 3
--tR61vgdxSzzv2FDycET2lt-OUZ1IW1GqA
Content-Disposition: form-data; name="other\";	
what=\"Something\""
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Value 4
--tR61vgdxSzzv2FDycET2lt-OUZ1IW1GqA--
