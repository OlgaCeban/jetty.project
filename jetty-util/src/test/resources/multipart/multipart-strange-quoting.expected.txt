Content-Type|multipart/form-data; boundary="tR61vgdxSzzv2FDycET2lt-OUZ1IW1GqA"
Parts-Count|4
Part-ContainsContents|and "I" quote|Value 1
Part-ContainsContents|and+%22I%22+quote|Value 2
Part-ContainsContents|value"; what="whoa"|Value 3
